CodingChallenges/CC_096_Visualizing_the_Digits_of_Pi/P5/sketch.js

```diff
@@ -0,0 +1,58 @@
+// Daniel Shiffman
+// http://codingtra.in
+// http://patreon.com/codingtrain
```

Comment:
```suggestion
```

FYI I'm no longer including the patreon link in source code, feels a little bit too much.